### ISSUE TITLE
关于环境版本的询问

### ISSUE THREAD

**Issue by Liucc2119** [THUKElab/DR-CSC#3]
您好，你不能麻烦您具体说一下每一个pip list的包的版本，因为后期pytorch_lightning改动较大，无法源代码无法运行，感谢您的帮助

**Comment by waterwaterrr** [comment]
python=3.9

Package                 Version
----------------------- --------------
absl-py                 1.3.0
aiohttp                 3.8.3
aiosignal               1.2.0
anyio                   3.6.2
argon2-cffi             21.3.0
argon2-cffi-bindings    21.2.0
astor                   0.8.1
asttokens               2.1.0
async-timeout           4.0.2
attr                    0.3.2
attrs                   21.2.0
Babel                   2.11.0
backcall                0.2.0
beautifulsoup4          4.11.1
bleach                  5.0.1
blessed                 1.19.1
cachetools              5.2.0
certifi                 2022.9.24
cffi                    1.15.1
charset-normalizer      2.1.1
click                   8.1.3
contourpy               1.0.6
cpm-kernels             1.0.11
cycler                  0.11.0
debugpy                 1.6.3
decorator               5.1.1
defusedxml              0.7.1
entrypoints             0.4
executing               1.2.0
fairscale               0.4.13
fastjsonschema          2.16.2
filelock                3.8.0
fire                    0.5.0
Flask                   2.2.3
fonttools               4.38.0
frozenlist              1.3.1
fsspec                  2022.10.0
future                  0.18.2
google-auth             2.14.0
google-auth-oauthlib    0.4.6
gpustat                 1.0.0
grpcio                  1.50.0
huggingface-hub         0.13.2
icetk                   0.0.4
idna                    3.4
importlib-metadata      5.0.0
ipykernel               6.17.0
ipython                 8.6.0
ipython-genutils        0.2.0
itsdangerous            2.1.2
jedi                    0.18.1
jieba                   0.42.1
Jinja2                  3.1.2
joblib                  1.2.0
json5                   0.9.10
jsonschema              4.17.0
jstyleson               0.0.2
jupyter_client          7.4.4
jupyter_core            4.11.2
jupyter-server          1.23.2
jupyterlab              3.5.0
jupyterlab-pygments     0.2.2
jupyterlab_server       2.16.3
kenlm                   0.0.0
kiwisolver              1.4.4
LAC                     2.1.2
Levenshtein             0.20.5
loguru                  0.6.0
lxml                    4.9.2
Markdown                3.4.1
MarkupSafe              2.1.1
matplotlib              3.6.2
matplotlib-inline       0.1.6
mistune                 2.0.4
multidict               6.0.2
nbclassic               0.4.8
nbclient                0.7.0
nbconvert               7.2.5
nbformat                5.7.0
nest-asyncio            1.5.6
networkx                2.8.8
notebook                6.5.2
notebook_shim           0.2.2
numpy                   1.24.2
nvidia-ml-py            11.495.46
oauthlib                3.2.2
openai                  0.27.0
OpenCC                  1.1.3
opt-einsum              3.3.0
ordered-set             4.1.0
packaging               21.3
paddle-bfloat           0.1.7
paddlepaddle            2.4.1
pandocfilters           1.5.0
parso                   0.8.3
pexpect                 4.8.0
pickleshare             0.7.5
Pillow                  9.3.0
pip                     22.2.2
prometheus-client       0.15.0
prompt-toolkit          3.0.32
protobuf                3.19.6
psutil                  5.9.3
ptyprocess              0.7.0
pure-eval               0.2.2
pyasn1                  0.4.8
pyasn1-modules          0.2.8
pycparser               2.21
pyDeprecate             0.3.1
Pygments                2.13.0
pyparsing               3.0.9
pypinyin                0.46.0
pyrsistent              0.19.2
python-dateutil         2.8.2
python-Levenshtein      0.20.5
pytorch-lightning       1.5.1
pytz                    2022.6
PyYAML                  6.0
pyzmq                   24.0.1
rapidfuzz               2.13.1
regex                   2021.10.8
requests                2.28.1
requests-oauthlib       1.3.1
rsa                     4.9
sacremoses              0.0.53
scikit-learn            1.1.3
scipy                   1.9.3
Send2Trash              1.8.0
sentencepiece           0.1.97
seqeval                 1.2.2
setuptools              59.5.0
six                     1.16.0
sniffio                 1.3.0
soupsieve               2.3.2.post1
speedtest-cli           2.1.3
stack-data              0.6.0
tensorboard             2.10.1
tensorboard-data-server 0.6.1
tensorboard-plugin-wit  1.8.1
tensorboardX            2.5.1
termcolor               2.2.0
terminado               0.17.0
threadpoolctl           3.1.0
tinycss2                1.2.1
tokenizers              0.13.2
tomli                   2.0.1
torch                   1.10.0+cu111
torchaudio              0.10.0+rocm4.1
torchmetrics            0.6.0
torchvision             0.11.0+cu111
tornado                 6.2
tqdm                    4.62.3
traitlets               5.5.0
transformers            4.26.1
typing_extensions       4.4.0
urllib3                 1.26.12
vpdb                    1.0.0
waitress                2.1.2
wcwidth                 0.2.5
webencodings            0.5.1
websocket-client        1.4.2
Werkzeug                2.2.2
wheel                   0.37.1
yacs                    0.1.8
yarl                    1.8.1
zipp                    3.10.0
